package.json

```diff
@@ -71,7 +71,7 @@
     "prop-types": "^15.7.2",
     "query-string": "^7.0.1",
     "react": "^17.0.2",
-    "react-contextmenu": "~2.14.0",
+    "@firefox-devtools/react-contextmenu": "~4.0.0",
```

Comment:
we can probably use ^ instead of ~ (not sure why this was the case before). See https://docs.npmjs.com/cli/v6/using-npm/semver#tilde-ranges-123-12-1 for reference.

Although in an app using yarn.lock this probably makes no big difference, so it's optional.

Reply:
Yeah, I was thinking the same as well when changes but decided to keep it. Still changed it now.